### ISSUE TITLE
How to locate where the dirtyDB data is stored?

### ISSUE THREAD

**Issue by pikrzysztof** [ether/etherpad-lite#3624]
I'm trying to migrate away from dirtyDB to postgres but I can't locate the file(s) which hold the data.

Can you please help me with constructing a grep expression or something else so I can extract my data?

`grep -RFi '"key":"pad:'` doesn't show any matches. Grepping for pad names or unique strings in pad contents doesn't bear any results as well.

**Comment by lpagliari** [comment]
I found [this on the wiki](https://github.com/ether/etherpad-lite/wiki/FAQ#how-can-i-prune-the-database-so-that-only-the-last-n-revisions-are-kept), maybe the file is on `var/dirty.db` or `/var/dirty.db`?

**Comment by pikrzysztof** [comment]
Unfortunately no. `/var/dirty.db` is empty and `var/dirty.db` has only a couple of lines so the data must be somewhere else (etherpad displays it fine). I've grepped my whole filesystem and I couldn't find it.

**Comment by muxator** [comment]
Hi @pikrzysztof, the Etherpad logs should contain, right after startup, the full path to the **dirty.db** file.

An example of the log contents would be:
```
[2019-07-11 22:28:50.673] [WARN] console - DirtyDB is used. This is fine for testing but not recommended for production. File location: /some/path/to/etherpad/etherpad-lite/var/dirty.db
```

This change was introduced in 435b2a4edf37 (released in **1.7.5**).

Side note on grepping: the `dirty.db` file is a text file made of json lines: the "normal" unix tools come a bit short for dealing with that.

There are two tools that are really handy to deal with files made of json lines, have a look at them if you want:
- [`jq`](https://stedolan.github.io/jq/): a swiss army knife, probably you already know it
- [`gron`](https://github.com/tomnomnom/gron): a little, intelligent tool, really useful for inspecting the structure of a json file, or representing it in a greppable format.
  Here's an example:
  ```
  $ ./gron --stream dirty.db   | grep "this is a test"
  json[691].val.atext.text = "this is a test\n\n";
  ```

**Comment by pikrzysztof** [comment]
@muxator Indeed they do. I've edited the code there to see what's going on and noticed that we're actually connecting to MySQL so please consider the question invalid.

Thank you for your help.
